File: 1. Conditional Problems/Conditional Problems in Python/p5.py
# write a python program to check whether a year is a leap year or not.

y = int(input("Enter the Year: "))

if (y % 4 ==0 and y %100 !=0) or y %400==0:
    print(f"{y} is a Leap Year.")
else:
    print(f"{y} is a not Leap Year.")

File: 1. Conditional Problems/Condional Problems in C/p5.c
// write a C program to check whether a y is a leap y or not.

#include <stdio.h>


int main() {


    int y;
    printf("Enter your Year: ");
    scanf("%d", &y);

    
    if ((y % 4 == 0 && y % 100 != 0) || (y % 400 == 0)) {
        printf("%d is a Leap Year.\n", y);
    } else {
        printf("%d is not a Leap Year.\n", y);
    }

    return 0;
}
